### PR TITLE
Drone k8s helm image

### PR DESCRIPTION
**Pull request by SimonForsman**

### CODE REVIEW

**Comment by dignite** [comment]
Is this the killer of the https://github.com/urbitassociates/drone-kubernetes-helm repo?

File: drone-kubernetes-helm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM mandrean/drone-kubernetes-helm:v0.3.0-k8s-v1.6-4-helm-v2.4.2
```
Comment (dignite) [inline drone-kubernetes-helm/Dockerfile:1]: Is there any risk this dependency changes or disappear? If so, can we avoid this dependency?

**Review by dignite — APPROVED** [review]
Great! 🌹 

Could we explain the use case of the docker image either as a comment in the `Dockerfile` or a `README.md`?

File: drone-kubernetes-helm/Dockerfile (hunk 1 above)
Comment (rodrigo-urbit) [reply drone-kubernetes-helm/Dockerfile:1]: Yes, maybe we can fork it. Could disappear, I believe

**Comment by dignite** [comment]
I'm guessing this docker image is no longer needed and should be deleted in favour of https://github.com/urbitassociates/drone-kubernetes-helm

Is that correct @SimonForsman @niklas-r ?

**Pull request closed without merging**